### PR TITLE
Dual normalization

### PR DESCRIPTION
**Pull request by victorventuri**
Implements normalization in dual framework, and provides an easy way to initialize an A-SOH-normalized dual UKF. Related to discussions in #59.

### CODE REVIEW

File: moirae/estimators/online/dual.py [hunk 1]
```diff
@@ -61,20 +66,41 @@ def __init__(self, transient_filter: BaseFilter, asoh_filter: BaseFilter) -> Non
...
+                :class:`~moirae.estimators.online.filters.distributions.MultivariateRandomDistribution` in
+                model-coordinates
+        """
         # We need to get the hidden states on both filters
         transient_hidden = self.trans_filter.hidden.model_copy(deep=True)
         asoh_hidden = self.asoh_filter.hidden.model_copy(deep=True)
-        return transient_hidden.combine_with(asoh_hidden)
+
+        # We need to transform them accordingly
+        transient_converted = transient_hidden.convert(conversion_operator=self.trans_filter.model.hidden_conversion)
```
Comment (WardLT) [inline moirae/estimators/online/dual.py:86]: Do we need the copy anymore if the the `convert` method produces a new instance of the distribution?

File: moirae/estimators/online/dual.py [hunk 2]
```diff
@@ -149,27 +197,51 @@ def initialize_unscented_kalman_filter(
...
-                           covariance_process_noise=transient_covariance_process_noise,
-                           covariance_sensor_noise=covariance_sensor_noise,
-                           **filter_args.get('transient', UKFTuningParameters.defaults()))
-        asoh_filter = UKF(model=asoh_wrapper,
-                          initial_hidden=asoh_hidden,
-                          initial_controls=initial_controls,
-                          covariance_process_noise=asoh_covariance_process_noise,
-                          covariance_sensor_noise=covariance_sensor_noise,
-                          **filter_args.get('asoh', UKFTuningParameters.defaults()))
+        trans_filter = UKF(
```
Comment (WardLT) [inline moirae/estimators/online/dual.py:228]: Tests which ensure we didn't have these numerous conversions backwards would be awesome

**Review by WardLT — COMMENTED** [review]
A few comments, since it sounds like you're still working on this

File: moirae/estimators/online/dual.py (hunk 1 above)
Comment (victorventuri) [reply moirae/estimators/online/dual.py:86]: You are right, no need to crowd memory. What do you think of this new version? (btw, I checked to make sure they run fine in the slow tests locally)

File: moirae/estimators/online/dual.py (hunk 2 above)
Comment (victorventuri) [reply moirae/estimators/online/dual.py:228]: Good idea! Will write one as well!

File: moirae/estimators/online/dual.py (hunk 1 above)
Comment (WardLT) [reply moirae/estimators/online/dual.py]: I was more pointing out the `model_copy` statements in the previous lines. You shouldn't need to copy and then convert, just convert.

**Review by WardLT — APPROVED** [review]
Looks great!

**Pull request merged**